src/molecules/feature/index.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import styled from "styled-components";
+import colors from "../../atoms/colors";
+import Copy, { Size as CopySize } from "../../atoms/copy";
+import Headline, { Level } from "../../atoms/headline";
+
+export interface FeatureProps {
+	/** @name Headline level */ headlineLevel: Level;
+	/** @name Headline */ headlineText: string;
+	/** @name Copy */ copyText: string;
+}
+
+const StyledFeature = styled.div`
+	display: inline-block;
+	padding: 60px 120px 60px 30px;
+	box-sizing: border-box;
+	color: ${colors.black.toString()};
+	text-align: left;
+	width: 50%;
+`;
+
+const StyledHeadline = styled(Headline)`
+	margin: 20px 0;
+`;
+
+const StyledCopy = styled(Copy)`
+	color: ${colors.grey70.toString()};
+`;
+
+const Feature: React.StatelessComponent<FeatureProps> = (props): JSX.Element => {
+	return (
+		<StyledFeature>
+			<svg width="40px" height="40px" viewBox="0 0 40 40" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please remove the `xmlns` property it's not necessary for the web.
I would suggest creating a **StyledSVG** component and define in CSS the width and height